main.js

```diff
@@ -0,0 +1,76 @@
+var snakewaterMontana = {
+    paleontologist: "Dr. Alan Grant",
+    depth: "10 meters",
+    specimen: "Velociraptor"
+  }
+
+  var guestOfHonor = snakewaterMontana.paleontologist;
+  var cleverGirl = snakewaterMontana.specimen; 
+
+  console.log (guestOfHonor);
+  console.log (cleverGirl);
+
+  nicaragua = {
+    depth: "200 meters",
+    annualBudget: 1500000,
+    specimens: [
+      "Tyrannosaurus Rex",
+      "Stegosaurous",
+      "Triceratops",
+      "Velociraptor"
+    ]
+  }
+annualBudget = annualBudget + 250000;
+console.log (annualBudget);
+  var specimens = ['Tyrannosaurus Rex', 'Stegosaurous','Triceratops', 'Velociraptor']
+  var nicaraguanSpecimens = nicaragua.specimens[0];
+  var nicaraguanSpecimens = nicaragua.specimens[1];
+  var nicaraguanSpecimens = nicaragua.specimens[2];
+  var nicaraguanSpecimens = nicaragua.specimens[3];
+
```

Comment:
I don't know what are you trying to do here but you just define a variable with the same name and assign it